lib/timber/integrations/phoenix_instrumenter.ex

```diff
@@ -138,19 +229,32 @@ defmodule Timber.Integrations.PhoenixInstrumenter do
     :ok
   end
 
+  #
+  # Utility
+  #
+
   @spec get_log_level(atom) :: atom
   defp get_log_level(default) do
     Timber.Config.phoenix_instrumentation_level(default)
   end
 
-  defp params(%{__struct__: :"Elixir.Plug.Conn.Unfetched"}), do: %{}
+  defp filter_params(%{__struct__: :"Elixir.Plug.Conn.Unfetched"}),
+    do: %{}
 
-  defp params(params) when is_list(params) or is_map(params) do
+  defp filter_params(params) when is_list(params) or is_map(params) do
     params
     |> Phoenix.Logger.filter_values()
     |> Enum.into(%{})
   end
 
   # Unknown type, convert to a blank map for now
-  defp params(_params), do: %{}
+  defp filter_params(_params),
+    do: %{}
+
+  defp try_atom_inspect(nil),
```

Comment:
Instead of matching on `nil`, we really want a catch for when we are inspecting something that is not an atom:

```elixir
def try_atom_inspect(t) when not is_atom(t) do
  nil
end
```